pom.xml

```diff
@@ -1,4 +1,8 @@
-<project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Do you have a ref to the previous discussions? I would be curious to see what the pros and cons of including the namespace and schema are for pom.xml?